package.json

```diff
@@ -2,9 +2,10 @@
   "name": "outline-client",
   "productName": "Outline",
   "scripts": {
+    "action": "bash ./scripts/run_action.sh",
+    "action:list": "find . -name '*.action.sh' | sed -E 's:./(.*).action.sh:\\1:' | grep -v 'scripts/run_action'",
     "clean": "rm -rf build www node_modules platforms/* plugins/*",
-    "do": "bash ./scripts/do_action.sh",
-    "actions": "find . -name '*_action.sh' | sed -E 's:./src/(.*)_action.sh:\\1:' | grep -v 'scripts/do_action'"
+    "lint:ts": "tslint --fix 'src/**/*.ts'"
```

Comment:
Having a lint command change code is unexpected. Can you remove the `--fix`?

Reply:
Ah, right